src/widgets/RealtimeEffectPanel.cpp

```diff
@@ -0,0 +1,711 @@
+                      : MovableControl(parent, winid, pos, size, style)
+      {
+         //Prevents flickering and paint order issues
+         MovableControl::SetBackgroundStyle(wxBG_STYLE_PAINT);
+
+         Bind(wxEVT_PAINT, &RealtimeEffectControl::OnPaint, this);
+
+         auto sizer = std::make_unique<wxBoxSizer>(wxHORIZONTAL);
+
+         //On/off button
+         auto enableButton = safenew ThemedButtonWrapper<wxToggleButton>(this, wxID_ANY, wxEmptyString, wxDefaultPosition, wxDefaultSize, wxNO_BORDER);
+         enableButton->SetBitmapLabelIndex(bmpEffectOff);
+         enableButton->SetBitmapPressedIndex(bmpEffectOn);
+         enableButton->SetBackgroundColorIndex(clrEffectListItemBackground);
+         enableButton->SetMinSize(enableButton->GetBitmapLabel().GetSize());
+         enableButton->Bind(wxEVT_TOGGLEBUTTON, &RealtimeEffectControl::OnEnableButtonClicked, this);
+         mEnableButton = enableButton;
+
+         //Central button with effect name
+         mChangeButton = safenew wxButton(this, wxID_ANY);
+         mChangeButton->SetMinSize(mEnableButton->GetMinSize());
+         mChangeButton->Bind(wxEVT_BUTTON, &RealtimeEffectControl::OnChangeButtonClicked, this);
+
+         //Show effect settings
+         auto optionsButton = safenew ThemedButtonWrapper<wxButton>(this, wxID_ANY, wxT(""), wxDefaultPosition, wxDefaultSize, wxNO_BORDER);
+         optionsButton->SetBitmapLabelIndex(bmpEffectSettingsNormal);
+         optionsButton->SetBitmapPressedIndex(bmpEffectSettingsDown);
+         optionsButton->SetBitmapCurrentIndex(bmpEffectSettingsHover);
+         optionsButton->SetBackgroundColorIndex(clrEffectListItemBackground);
+         optionsButton->SetMinSize(optionsButton->GetBitmapLabel().GetSize());
+         optionsButton->Bind(wxEVT_BUTTON, &RealtimeEffectControl::OnOptionsClicked, this);
+         
+         auto dragArea = safenew wxStaticBitmap(this, wxID_ANY, theTheme.Bitmap(bmpDragArea));
+         dragArea->Disable();
+         sizer->Add(
+            dragArea,
+            wxSizerFlags().Center().Border(wxLEFT, 5)
+         );
+         sizer->Add(
+            mEnableButton,
+            wxSizerFlags().Center().Border(wxLEFT, 5)
+         );
+         sizer->Add(
+            mChangeButton,
+            wxSizerFlags().Center().Border(wxLEFT, 5).Proportion(1)
+         );
+         sizer->Add(
+            optionsButton,
+            wxSizerFlags().Center().Border(wxLEFT | wxRIGHT, 5)
+         );
+
+         SetSizer(sizer.release());
+      }
+
+      void SetEffect(const std::shared_ptr<Track>& track, int effectIndex)
+      {
+         mTrack = track;
+
+         auto& effects = RealtimeEffectList::Get(*track);
+         assert (effectIndex >= 0 && effectIndex < effects.GetStatesCount());
+
+         mEffectState = effects.GetStateAt(effectIndex);
+         auto desc = PluginManager::Get().GetPlugin(mEffectState->GetID());
+
+         mChangeButton->SetLabel(desc->GetSymbol().Translation());
```

Comment:
I see uses of `Translation` here and elsewhere when building buttons.  That means one of these windows should be a `PrefsListener` and recreate those labels, in case the user changes the language in Interface preferences.